services/QuillLMS/spec/queries/adapters/csv/admin_premium_data_export_spec.rb

```diff
@@ -5,6 +5,14 @@
 describe Adapters::Csv::AdminPremiumDataExport do
   subject { described_class }
 
+  describe '#format_cell' do
+    it { expect(described_class.format_cell(:completed_at, DateTime.new(2020,1,1))).to eq('2020-01-01')}
+    it { expect(described_class.format_cell(:activity_pack, 'foo')).to eq('foo')}
+    it { expect(described_class.format_cell(:timespent, 61)).to eq(1)}
+    it { expect(described_class.format_cell(:score, 0.66777)).to eq('67%')}
+    it { expect(described_class.format_cell(:not_a_special_case, 1.5)).to eq(1.5)}
```

Comment:
I always appreciate when arbitrary values like this help me understand what the test case is.  Nice!